cypress/support/validations/schemaValidation.js

```diff
@@ -152,40 +112,35 @@ Cypress.Commands.add(
   * @param {Object} response - JSON response string
   * @param {string} methodOrEvent - String containing the method/event in the format "<Module.Method>" or "<Module.Event>"(Ex: accessibility.closedCaptionsSettings, 'accessibility.onClosedCaptionsSettingsChanged')
   * @param {*} params - API params
-  * @param {string} sdkVersion - SDK version
   * @param {string} schemaType - schema type determines which schema should fetch result/error
   * @example
   * validateSchema('"accessibility.closedCaptionsSettings",{"enabled":true,"styles":{"fontFamily":"Monospace sans-serif","fontSize":1,"fontColor":"#ffffff","fontEdge":"none","fontEdgeColor":"#7F7F7F","fontOpacity":100,"backgroundColor":"#000000","backgroundOpacity":100,"textAlign":"center","textAlignVertical":"middle"}}'
    {}, "0.17.0", "result")
   */
-Cypress.Commands.add(
-  'validateSchema',
-  (response, methodOrEvent, params, sdkVersion = null, schemaType) => {
-    cy.getSchema(methodOrEvent, params, sdkVersion, schemaType).then((schemaMap) => {
-      if (schemaMap) {
-        return validator.validate(response, schemaMap);
-      } else {
-        cy.log(`Failed to fetch schema, validateSchema`).then(() => {
-          assert(false, 'Failed to fetch schema, validateSchema');
-        });
-      }
-    });
-  }
-);
+Cypress.Commands.add('validateSchema', (response, methodOrEvent, params, schemaType) => {
+  cy.getSchema(methodOrEvent, params, schemaType).then((schemaMap) => {
+    if (schemaMap) {
+      return validator.validate(response, schemaMap);
+    } else {
+      cy.log(`Failed to fetch schema, validateSchema`).then(() => {
+        assert(false, 'Failed to fetch schema, validateSchema');
```

Comment:
Does this occur during "I validate" or during "I call"?

We want calls to come back and get stored even if they don't adhere to the schema. They should only fail during the validation step. If this is only called during the validation step please ignore this comment.

Reply:
While testing, I have observed above failure during invoke glue as well since above function is internally called for both cases. To address above comment, I have added a new flag 'isValidation' to determine if above function is called as part of invocation/validation glue and only execute above logic to fail if 'isValidation' is set to true